packages/server/lib/modes/run.ts

```diff
@@ -645,7 +647,14 @@ function waitForTestsToFinishRunning (options: { project: Project, screenshots:
         quiet,
         ffmpegChaptersConfig,
       )
-      .catch(warnVideoRecordingFailed)
+      .catch((err) => {
+        videoCaptureFailed = true
+        warnVideoRecordingFailed(err)
+      })
+    }
+
+    if (videoCaptureFailed) {
```

Comment:
does the `await` above also wait for the catch? I would think the `if` would fun first then the possible catch. Wonder if it is a better/more readable to cut this over to a try/catch/finally?

Reply:
@AtofStryker Sorry, which `await`? Theres a couple things happening in the after test run handler.